src/content/ui/adaptive-responsive/index.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Overview
+description: >
+  It's important to create an app,
+  whether for mobile or web,
+  that responds to size and orientation changes
+  and maximizes the use of each platform.
+short-title: Adaptive
+---
+
+One of Flutter's primary goals is to create a framework
+that allows you to develop apps from a single codebase
+that look and feel great on any platform.
+
+This means that your app might appear on screens of
+many different sizes, from a watch, to a foldable
+phone with two screens, to a high definition monitor.
+And your input device might be a physical or
+virtual keyboard, a mouse, a touchscreen, or
+any number of other devices.
+
+Two terms that describe these design concepts
+are _adaptive_ and _responsive_. Ideally,
+you'd want your app to be _both_ but what,
+exactly, does this mean?
+
+## What is responsive vs adaptive?
+
+An easy way to think about it is that responsive design
+is about fitting the UI _into_ the space and
+adaptive design is about the UI being _usable_ in
+the space.
+
+So, a responsive app adjusts the placement of design
+elements to _fit_ the available space. And an
+adaptive app selects the appropriate layout and
+input devices to be usable _in_ the available space.
+For example, on the current device,
+should the UI use bottom navigation or
+side-panel navigation?
+
+:::note
+Often adaptive and responsive concepts are
+collapsed into a single term. Most often,
+_adaptive design_ is used to refer to both
+adaptive and responsive.
+:::
+
+This section covers various aspects of adaptive and
```

Comment:
@TytaniumDev I do you think we should include the 3 steps to build an adaptive design in this overview page?

Reply:
Yeah I think we should include it here if this section grows to add examples, and if not we should definitely add it as one of the links below.

I think the 3 steps are mostly relevant as either a "building a new adaptive UI" or "updating a UI to be adaptive". I could see either being on their own page.

Reply:
@reidbaker @TytaniumDev, I believe I've largely addressed this.